src/routes/index.jsx

```diff
@@ -0,0 +1,34 @@
+import { createBrowserRouter } from "react-router-dom";
+
+import App from "../App";
+import ItemsPage from "@pages/items-page";
+import LandingPage from "@pages/landing-page";
+import BoardsPage from "@pages/boards-page";
+import AddItemPage from "@pages/add-item-page";
+
+const router = createBrowserRouter([
+  {
+    path: "/",
+    element: <App />,
+    children: [
+      {
+        path: "",
+        element: <LandingPage />,
+      },
```

Comment:
💊 제안
index 속성을 이용해 시작페이지라는 것을 더 명확하게 알 수 있게 하시는 것을 추천드립니다~

```suggestion
      {
        index: true,
        element: <LandingPage />,
      },
```